failed: true
msg: 'invalid literal for int() with base 10: ''hostname TestRouter\n!\nno ip domain
  lookup\nip domain name example.com\n!\nip domain retry 3\nip domain timeout 2\nip
  name-server 2001:558:FEED::1\nip name-server 8.8.8.8\nip name-server 75.75.75.75'
exception: "  File \"/usr/share/ansible/collections/ansible_collections/ansible/netcommon/plugins/module_utils/network/common/facts/facts.py\",
  line 126, in get_network_resources_facts\n    inst.populate_facts(self._connection,
  self.ansible_facts, data)\n  File \"/usr/share/ansible/collections/ansible_collections/cisco/ios/plugins/module_utils/network/ios/facts/lacp/lacp.py\",
  line 62, in populate_facts\n    lacp_obj = self.render_config(self.generated_spec,
  data)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/share/ansible/collections/ansible_collections/cisco/ios/plugins/module_utils/network/ios/facts/lacp/lacp.py\",
  line 87, in render_config\n    config[\"system\"][\"priority\"] = int(conf.split(\",\")[0])\n
  \                                  ^^^^^^^^^^^^^^^^^^^^^^^\n"
changed: false
